### DISCUSSION TITLE
[core/not-initialized] Firebase has not been correctly initialized.

### DISCUSSION THREAD

**Discussion by samuelgauer** [firebase/flutterfire#13494] [Q&A]
I'm using Flutter 3.24.3 with firabase_core 3.6.0.

in main.dart when I try to initialize Firebase raise this error:

[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: [core/not-initialized] Firebase has not been correctly initialized.
           
           Usually this means you've attempted to use a Firebase service before calling `Firebase.initializeApp`.
           
           View the documentation for more information: https://firebase.google.com/docs/flutter/setup
           
           #0      MethodChannelFirebase.initializeApp (package:firebase_core_platform_interface/src/method_channel/method_channel_firebase.dart:120:9)
           <asynchronous suspension>
           #1      Firebase.initializeApp (package:firebase_core/src/firebase.dart:66:31)
           <asynchronous suspension>

I just check GoogleService-Infor.plist and looks ok.

Thanks

**Comment by Akhileshrao89** [comment]
bro you did you get the solution

**Comment by sjonchhe** [comment]
any solutions for this?
